### ISSUE TITLE
Scroll directly to the comment when clicking notification

### ISSUE THREAD

**Issue by alejandro5042** [CodeHubApp/CodeHub#627]
On a long comment thread, scrolling to the bottom is tedious to find the comment you were notified about.

Suggestion: Either have a scroll to bottom button or just direct the user to the bottom to begin with (or last unread).

**Comment by thedillonb** [comment]
Yeah comments are definitely a hassle right now. There are a few ideas floating around such as re-ordering the list from most recent at the top to less recent or, keep the current order, show the recent 30 or so, and collapse the past comments in a button at the top which can be tapped to expand.
